links/src/jsMain/kotlin/com/splendo/kaluga/links/manager/PlatformLinksHandler.kt

```diff
@@ -22,7 +22,7 @@ actual class PlatformLinksHandler : LinksHandler {
         TODO("Not yet implemented")
     }
 
-    override fun extractQueryAsList(url: String): List<NameValue> {
+    override fun extractQueryAsList(url: String): NameValue {
```

Comment:
```suggestion
    override fun extractQueryAsList(url: String): ParametersNameValue {
```

And check imports (should be same way as for jvm)?

Reply:
c2adaea

Reply:
c2adaea